### DISCUSSION TITLE
Valitdation of FHIR Bundle

### DISCUSSION THREAD

**Discussion by beauleblond** [microsoft/FHIR-Converter#264] [Q&A: Liquid Engine]
Hey all, I'm getting a lot of "Unable to resolve resource" or "unknown code" errors in the validation of converted data. Is this just due to the sample data provided being "made up", or is there another reason for this happening? Any insight would be helpful.

**Comment by ginalee-dotcom** [comment]
Could you provide more information on your full validation results and perhaps the sample data you used as well? So that we can better understand what is the resource or the code that is not able to be resolved? Thanks!

**Comment by beauleblond** [comment]
I was using the sample CCDA’s given in the GitHub. I think my main problem
is how I can tell if a whole bundle is valid without going through resource
by resource. I am a recently hired intern so I’m not quite sure how this
all works, so forgive me if I say something wrong. If you have time for a
chat on how the converter works as well that would be much appreciated but
no worries if not. Thank you!
